pkg/apis/gcp/v1alpha1/common_types.go

```diff
@@ -0,0 +1,13 @@
+package v1alpha1
+
+// LegalEntity contains Red Hat specific identifiers to the original creator the clusters
+type LegalEntity struct {
+	Name string `json:"name"`
+	ID   string `json:"id"`
+}
+
+// NamespacedName contains the name of a secret and its namespace
```

Comment:
We use this as generic name of any Kubernetes object, right? Like, as a reference to a `ProjectClaim`, so I assume it's not just secret.